books/RayTracingTheNextWeek.html

```diff
@@ -1097,47 +1118,64 @@
         bool front_face;
         ...
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    [Listing [hit-record-uv]: <kbd>[hittable.h]</kbd> Adding U,V coordinates to the `hit_record`]
+    [Listing [hit-record-uv]: <kbd>[hittable.h]</kbd> Adding $u,v$ coordinates to the `hit_record`]
 
-We will also need to compute $(u,v)$ texture coordinates for hittables.
+We will also need to compute $(u,v)$ texture coordinates for a given point on each type of
+`hittable`.
 
 
 Solid Textures: A Checker Texture
 ----------------------------------
-<div class='together'>
-We can create a checker texture by noting that the sign of sine and cosine just alternates in a
-regular way, and if we multiply trig functions in all three dimensions, the sign of that product
-forms a 3D checker pattern.
+The next simplest texture map type is the solid texture, which depends only on the position of each
+point in 3D space. You can think of a solid texture (of color) as coloring space itself, visible on
+the object surface. For this reason, the object can move through the colors of the texture as it
+changes position, though usually you would want to fix the relationship between the object and the
+solid texture.
+
+To accomplish the checkered pattern, we'll first compute the floor of each component of the input
+point. We could truncate the coordinates, but that would pull values toward zero, which would give
```

Comment:
you might want to reinforce that the input point is $(u,v)$

Reply:
Yeah, that triggered me to write an entire paragraph that was missing.